setup.cfg

```diff
@@ -1,9 +1,121 @@
-[versioneer]
-VCS = git
-style = pep440-post
-versionfile_source = ophyd/_version.py
-versionfile_build = ophyd/_version.py
-tag_prefix = v
+[metadata]
```

Comment:
Can any of this be moved to and used from `pyproject.toml`?  There is a [recommendation](https://pypi.org/project/setuptools-scm/) to deprecate `setup.cfg`:

> usage in setup.cfg is considered deprecated, please use pyproject.toml whenever possible

Reply:
I think most things can read from `pyproject.toml`. [tox](https://tox.wiki/en/latest/example/basic.html#pyproject-toml-tox-legacy-ini) just embeds the ini under a specific key, which is a bit ugly, but flake8 [won't add support for pyproject.toml](https://github.com/PyCQA/flake8/issues/234#issuecomment-1206730688) until pip changes its behaviour on the presence of pyproject.toml. I'll try moving some things and see if it works

Reply:
Actually, as the `[tool.setuptools]` table in pyproject.toml [is still in beta](https://setuptools.pypa.io/en/latest/userguide/quickstart.html#beta) I think it's best to leave things in setup.cfg until they have stabilized.

Reply:
> There is a [recommendation](https://pypi.org/project/setuptools-scm/) to deprecate `setup.cfg`:

It looks like this recommendation is from setuptools-scm, and we already put its configuration in pyproject.toml